install.sh

```diff
@@ -5,12 +5,13 @@
 set -e
 
 if [ $# -eq 0 ]; then
-  version="v0.2.6"
+  version="v0.2.7"
```

Comment:
I Always assumed the python installer did auto detection.

Reply:
The python one does. This one has it hardcoded for now.